pkg/agent/baker_test.go

```diff
@@ -142,6 +142,17 @@ var _ = Describe("Assert generated customData and cseCmd", func() {
 			configUpdator(config)
 		}
 
+		// CSE
+		cseCommand := baker.GetNodeBootstrappingCmd(config)
```

Comment:
GetNodeBootstrappingPayload (customData) cleaned the input, then used by GetNodeBootStrappingCmd (CSE)

flipping the order and showing the testdata doesn't change to add --dyamic-config-dir is the validation of the change

previous to this, the flag was still present in CSE.